en/admin/admin_utils.rst

```diff
@@ -817,7 +817,7 @@ The following shows [options] available with the **cubrid compactdb** utility.
 
 .. option:: -i, --input-class-file=FILE
 
-    You can specify an input file name that contains the table name with this option. Write one table name in a single line; invalid table name is ignored. Note that you cannot specify the list of the table names after a database name in case of you use this option. If you use this option on client/server mode, It is skipped freeing objects such as catalog, delete files and tracker etc.
+    You can specify an input file name that contains the table name with this option. Write one table name in a single line; invalid table name is ignored. Note that you cannot specify the list of the table names after a database name in case of you use this option. If you use this option on client/server mode, It skips securing space taken by objects such as catalog, delete files and tracker, etc.
```

Comment:
'It' need to be changed to lower case 'it'

Reply:
thank you. i modified it.